docs/book/toc.md

```diff
@@ -24,6 +24,7 @@
   * [Manage environments](user-guide/advanced-guide/manage-environments.md)
   * [Containerize your pipeline](user-guide/advanced-guide/containerize-your-pipeline.md)
   * [Connect your git repository](user-guide/advanced-guide/connect-your-git-repository.md)
+  * [Use the Client](./user-guide/advanced-guide/client.md)
```

Comment:
I maybe would bump this up a bit, after 'visualise artifacts' maybe? It's quite important.

Reply:
Yeah makes sense, actually that's exactly where I intuitively wanted to put it as well initially 👍